### PR TITLE
[Frontend] Tailwind

### PR DESCRIPTION
**Pull request by CamronLDNF**
## Description
Sets up Tailwind framework

## PivotalTracker
[ Set up Tailwind](https://www.pivotaltracker.com/story/show/163498914)

## Tasks
- Install Tailwind React UI framework
- Install Tailwind

## Screenshots 
skarmavbild 2019-01-31 kl 15 02 20

### CODE REVIEW

File: client/we_meet_client/src/App.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+
 
 class App extends Component {
   render() {
     return (
-      <div>
-        <h1>WeMeet</h1>
-        <Events />
-      </div>
+      <>
```
Comment (shahin1290) [inline client/we_meet_client/src/App.js:11]: I am seeing this empty tag for first time. Is that possible?

**Comment by Jon1969Edwards** [comment]
Other then Shahin's comment I don't see any problems.

File: client/we_meet_client/src/App.js (hunk 1 above)
Comment (tochman) [reply client/we_meet_client/src/App.js:11]: This is a shorthand for `React.Fragment`

Comment (shahin1290) [reply client/we_meet_client/src/App.js:11]: Cool

**Pull request merged**